phpBB/includes/functions.php

```diff
@@ -2200,7 +2200,7 @@ function confirm_box($check, $title = '', $hidden = '', $html_body = 'confirm_bo
 		return false;
 	}
 
-	// re-add sid / transform & to &amp; for user->page (user->page is always using &)
+	// re-add sid / transform & to &amp; for user->page (uhttps://github.com/rubencm/phpbb/pull/new/ticket/15958ser->page is always using &)
```

Comment:
errors in the url, should be:
`// re-add sid / transform & to &amp; for user->page (https://github.com/rubencm/phpbb/pull/new/ticket/15958 user->page is always using &)`